src/client/fox.js

```diff
@@ -702,20 +702,13 @@ function randomizeColors( colors, variations ) {
 
 };
 
-var loader = new THREE.JSONLoader(true);
-var foxGeometry = null;
-loader.load('/content/fox.json', function(geo) {
-    console.log('Fox geometry loaded');
-    foxGeometry = geo;
-});
-
 /**
  * Represents a fox.
  */
-function Fox(direction) {
+function Fox(direction, geometry) {
```

Comment:
I think we should change this name later, since it covers more than just the fix now :smiley: